### PR TITLE
Important heretic spell rebalancing

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Nerfs
- Furious steel cooldown: 30s -> 60 seconds (when ascended: 10s -> 30s)
- Furious steel: Now affected by antimagic
- Cleave cooldown: 40s -> 45s
- Cleave range: 9 tiles -> 4 tiles
- Cleave wound: Now has natural clotting, changing the amount of blood loss from inf -> ~40% 
- Blood siphon range: 9 tiles -> 6 tiles
- Void Pull: Now affected by antimagic
- Void Phase: Now affected by antimagic

Buffs
- Void Blast cooldown: 60s -> 30s

Other
- Rust Formation now has a "distinct" icon
- Void Blast now has a "distinct" icon

## Why It's Good For The Game

A lot of these spells were extremely oppressive, and made it pretty much a joke to get away with anything. 
They were no-brainer choices, and as a result no one really pathed into anything else but these. 

- Furious Steel: 
   - Now that blade heretics have "realignment" in their repertoire, which offers them another counter for being hit by disablers or batons, this spell doesn't need to have such an insanely high uptime. The spell should be used for initiating and obtaining the lead in a combat, instead of having nigh-invulnerability for most periods. 
   - Additionally, antimagic protection was kind of missing, which was partially an oversight of it not being a `/magic` projectile.
 
- Cleave:
   - Cleave was by far the most absurd ability available bar none. This spell was guaranteed death in 30 seconds if the target had no way to stop the bloodflow immediately. AND it could be casted from across the screen. This brings cleave's range into midrange between you and the target, giving a lot more opportunity to be aware for the victim. 
   - Critical bleed wounds had a negative clotting rate, meaning that prior you would bleed to 0% from cleave if you didn't stop it. Not very fun, so with the default clotting rate it now stops at 60% blood flow - enough to be lethal if untreated, but doesn't completely tap you out
   - **Alternatives**: 
      - Keep the no clotting, make it a pure melee / touch spell. 
      - Reduce the cooldown, make it a projectile
      - Change it to be like a cool scythe attack that comes out of the caster and does a sweep

- Blood Siphon: 
   - This was primarily done to slot in better with Cleave's range decrease, encouraging more close range combat between the two. Getting point clicked from across the screen isn't fun.

- Void Pull and Phase:
   - Largely done for consistency. These are spells which cause damage, so anti-magic should stop the damage from the spells. 

- Void Blast
   - I have no idea why I made the cooldown so high on this, 1 minute made it almost worthless. 

TLDR: Instakill click spells from across the screen bad, invulnerability bad

## Changelog

:cl: Melbert
balance: Heretic: Furious Steel's cooldown has been doubled (30s -> 60s), and abides by antimagic
balance: Heretic: Cleave's cooldown has increased by 5s, range has been decreased to 4 tiles, and wound applied now has natural clotting
balance: Heretic: Blood Siphon's range has been decreased to 6 tiles
balance: Heretic: Void Pull and Phase abide by antimagic
balance: Heretic: Halved Void Blast's cooldown to 30s
qol: Heretic: Void Blast and Rust Formation now have distinct icons 
/:cl:

### CODE REVIEW

**Comment by YuliceTheDruid** [comment]
Local man abuses Blade Heretic for 1 (One) round, immediately gets it slapped with a host of nerfs. (Granted a lot of these were probably needed anyhow given how fucking powerful that combo was)

**Comment by Overuseddart** [comment]
I feel as if the ascended cooldown should still be the same, they are supposed to be incredibly powerful. (I still think we need to buff all the ascensions to match up to flesh)

**Comment by saprasam** [comment]
> I feel as if the ascended cooldown should still be the same, they are supposed to be incredibly powerful. (I still think we need to buff all the ascensions to match up to flesh)

agreed, i feel like ascended heretics could definitely have pre nerf spells
for fun, of course, not because i want to murder the entire station or anything

**Comment by datorangebottle** [comment]
I honestly don't think heretic needs nerfs. If these spells stand above and beyond the rest of heretic's abilities, perhaps the others should be buffed to compensate.

**Comment by saprasam** [comment]
>   * Now that blade heretics have "realignment" in their repertoire, which offers them another counter for being hit by disablers or batons, this spell doesn't need to have such an insanely high uptime. The spell should be used for initiating and obtaining the lead in a combat, instead of having nigh-invulnerability for most periods.

i feel like it's worth mentioning that realignment has never saved me at all and i'm not even sure if it can even save you from being permastunned by 1 guy with a baton. would test, but local servers aren't loading tgui for me so lol, lmao

**Comment by KittyNoodle** [comment]
i think the primary problem with cleave's blood loss is that blood loss in general is very not good and suffers a far worse version of what radiation had.
also adding an arbitrary range onto spells is really bad considering that in no way is the cooldown telegraphed or explained to the player, and it also runs the cooldown for the spell even if the cast was not successful.

**Comment by Overuseddart** [comment]
Maybe you could change it so on ascending, the spells are buffed to their pre-nerfed state? Certain ascended heretics benefit greatly from these spells.

**Comment by umoi0** [comment]
Isn't Furious steel a summon magic that summon knifes? that mean the knifes are non-magic exist when summoned, guess it should not affected by antimagic tho

**Comment by KittyNoodle** [comment]
> > also adding an arbitrary range onto spells is really bad considering that in no way is the cooldown telegraphed or explained to the player, and it also runs the cooldown for the spell even if the cast was not successful.
> 
> a. cooldown is telegraphed very blatantly to the player b. it doesn't run the cooldown if you fail to cast due to range
> 
> jesse what are you talking about

Phrased this badly but my complaint is that that the range is not telegraphed to the player in any way. This was less of a problem before because the spells were either click on someone adjacent to you, or click on anyone on your screen. What this will end up doing is making using these spells in any combat scenario useless, as having to, in addition to clicking on the player’s hit box, now must calculate the distance in tiles(a completely non-intuitive way of visually determining distance) while both you and whoever you are trying to hit are rapidly moving
